packages/react/vite.config.ts

```diff
@@ -11,6 +11,16 @@ export default defineConfig((conf) => ({
     setupFiles: ["./vitestSetup.ts"],
   },
   plugins: [react()],
+  // used so that vitest resolves the core package from the sources instead of the built version
```

Comment:
Is this necessary here? In PR #418 I saw you removed it, not sure if it's the same case here

Reply:
yes, I readded it with the comment